i18n/id.toml

```diff
@@ -0,0 +1,195 @@
+# Terjemahan untuk Bahasa Indonesia
+# https://gohugo.io/content-management/multilingual/#translation-of-strings
+
+# === baseof ==
+[backToTop]
+other = "Kembali ke Atas"
+
+[viewComments]
+other = "Lihat Komentar"
+# === baseof ==
+
+# === Post ===
+[posts]
+other = "Pos"
+# === Post ===
+
+# === Taxonomy ===
+[allSome]
+other = "Semua {{ .Some }}"
+
+[tag]
+other = "Label"
+
+[tags]
+other = "Label"
+
+[category]
+other = "Kategori"
+
+[categories]
+other = "Kategori"
+# === Taxonomy ===
+
+# === Pagination ===
+[more]
+other = "Lainnya"
+# === Pagination ===
+
+# === partials/header.html ===
+[selectLanguage]
+other = "Pilih Bahasa"
+
+[switchTheme]
+other = "Ganti Tema"
+# === partials/header.html ===
+
+# === partials/footer.html ===
+[poweredBySome]
+other = "Ditenagai oleh {{ .Hugo }} | Tema - {{ .Theme }}"
```

Comment:
`Didukung oleh....`

Reply:
IMO, "Didukung oleh" is more precisely translated as "Supported by". I remember WordPress also translated "Powered by" to "Ditenagai oleh".

Reply:
> IMO, "Didukung oleh" is more precisely translated as "Supported by". I remember WordPress also translated "Powered by" to "Ditenagai oleh".

1. As far is i know, there are no references to use conjuction with **_di-i_**, so _Didukung_ is a wrong word.
2. The translation is come from **_Powered_** followed with _**by**_, so the exact translation is _**Didukung oleh**_ or _**Dipersembahkan oleh**_

Reply:
**Dipersembahkan oleh** seems good for this translation, but I found another option, **Diberdayakan oleh**. What do you think?